src/scss/qld-print.scss

```diff
@@ -264,7 +269,7 @@
     }
   
     /* Hide elements not needed for printing */
-    .navbar, footer, .sidebar {
+    .navbar, footer, .sidebar, footer.qld-footer.dark {
```

Comment:
Wouldn't this work on just footer? or does it need footer dark?

Reply:
@MitchAnderson93 Yeah I thought just footer would work too, but it apparently doesn't...